app/jobs/appointment_notification/worker.rb

```diff
@@ -6,14 +6,33 @@ class AppointmentNotification::Worker
 
   DEFAULT_LOCALE = :en
 
+  class Metrics
```

Comment:
Going to leave this as an inner class until I have a use for this sort of thing elsewhere, which will probably be soon.

Reply:
never mind, extracted already.